forms-flow-web/src/components/Modeller/Editors/BpmnEditor/index.js

```diff
@@ -324,12 +319,6 @@ export default React.memo(
         </div>
 
         <div>
-          {MULTITENANCY_ENABLED ? (
```

Comment:
we would still need this if this feature is enabled.

Reply:
now we the user can create the new task as private by default and can create a public one changing the env. so should we need to add this feature?